### PR TITLE
fix net amount calculation

### PR DESCRIPTION
**Pull request by kewde**

### CODE REVIEW

**Comment by kewde** [comment]
@allienworks this should fix it. I didn't test this tho, but the problem was quite clear.

File: src/app/wallet/shared/transaction.model.ts [hunk 1]
```diff
@@ -54,6 +54,22 @@ export class Transaction implements Deserializable {
     return this.txid + this.getAmount() + this.category;
   }
 
+  public getNetAmount() {
+    const amount : number = +this.getAmount();
+    if (amount < 0) { // sent
+      return amount + this.fee;
```
Comment (allienworks) [inline src/app/wallet/shared/transaction.model.ts]: Sorry, but `ng serve` complains about:
```
ERROR in /srv/http/partgui/src/app/wallet/shared/transaction.model.ts (60,28): Property 'fee' does not exist on type 'Transaction'.
ERROR in /srv/http/partgui/src/app/wallet/shared/transaction.model.ts (62,30): Property 'fee' does not exist on type 'Transaction'.
```

Comment (allienworks) [reply src/app/wallet/shared/transaction.model.ts]: When fixed, Travis will stop complaining.

**Comment by kewde** [comment]
#192

**Pull request merged**